### PR TITLE
Web UX parity: file preview, document graph, auto run, and mobile enhancements

### PR DESCRIPTION
**Pull request by chr1syy**
## Summary
- **File Preview**: Full syntax-highlighted file preview with markdown rendering, image support, and large file truncation via WebSocket `read_file` handler
- **Document Graph**: List-based link explorer for navigating document relationships, wired through WebSocket `get_document_graph` handler
- **Auto Run**: Inline document preview with task summaries, folder grouping with path display, and race condition fixes for rapid document selection
- **File Explorer**: File-type icons, indent guides, header toolbar, selection/hover styling, and long-press context menus
- **Image Posting**: Full pipeline for attaching and sending images from the web UI through WebSocket to agents
- **Worktree Creation**: Remote worktree creation UI with context menus and IPC pipeline
- **Mobile UX**: Unread-only filter, dynamic chat input sizing, system prompt inclusion for web-spawned agents

## Test plan
- [x] TypeScript type checks pass (both web and main configs)
- [x] Web server factory tests pass (307 tests)
- [x] Web mobile tests pass (2667 tests)
- [x] ESLint and Prettier clean on all changed files
- [ ] Manual verification of file preview, document graph, and auto run tabs in web UI
- [ ] Manual verification of image posting flow end-to-end

🤖 Generated with [Claude Code](https://claude.com/claude-code)

## Summary by CodeRabbit

* **New Features**
  * Added unread state indicators for AI tabs to highlight tabs with new content
  * Enhanced Auto Run document cards with task progress visualization, displaying completion percentages and status indicators

* **Bug Fixes**
  * Corrected Auto Run directory path reference in empty-state messaging

### CODE REVIEW

File: src/web/mobile/AutoRunDocumentCard.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+export interface DocumentCardProps {
+	document: AutoRunDocument;
+	onTap: (filename: string) => void;
+}
+
+export function DocumentCard({ document: doc, onTap }: DocumentCardProps) {
+	const colors = useThemeColors();
+	const progress = doc.taskCount > 0 ? Math.round((doc.completedCount / doc.taskCount) * 100) : 0;
+
```
[the remark this answers is not in the document]
Comment (chr1syy) [reply src/web/mobile/AutoRunDocumentCard.tsx:20]: Fixed in 1961e5ab0 — progress is now clamped to 0–100 via `Math.max(0, Math.min(100, ...))`. Added tests covering overflow and underflow edge cases.

File: src/web/mobile/RightDrawer.tsx [hunk 2]
```diff
@@ -659,59 +668,199 @@ function HistoryTabContent({
...
+	}, [sessionId, loadDocuments]);
+
+	const handleStop = useCallback(async () => {
+		triggerHaptic(HAPTIC_PATTERNS.interrupt);
+		setIsStopping(true);
+		const success = await stopAutoRun(sessionId);
+		if (!success) {
+			setIsStopping(false);
+		}
+	}, [sessionId, stopAutoRun]);
```
[the remark this answers is not in the document]
Comment (chr1syy) [reply src/web/mobile/RightDrawer.tsx:723]: Fixed in 1961e5ab0 — handleStop now wraps stopAutoRun in try/catch, resets isStopping on error, and re-throws for Sentry. Added tests verifying the flag resets on both failure return and thrown errors.
